script.js

```diff
@@ -1,12 +1,247 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
+let allShows;
```

Comment:
there is no need for this to be a global variable, since it is only used inside one function `setup()`

Reply:
Thank you Jonathan, i think i will use that on the shows listing so probably i'm going to leave it there, but i sure need some practice on removing what is not necessary

Reply:
yes that is fine